ops/Makefile

```diff
@@ -46,6 +46,10 @@ init-%: .valid-env-%
 	terraform -chdir=$*/persistent init
 	terraform -chdir=$* init
 
+plan-%: .valid-env-% api.tfvars
+	terraform -chdir=$*/persistent plan -lock=false
```

Comment:
We shouldn't need to avoid locking here (at least I don't think), because multiple GH Actions using the same workflow will queue themselves instead of running concurrently.

Reply:
That's a good callout! The reason why I added this was because I experienced a situation where one of the plan actions caused by the GH Action run locked the state, then was cancelled, which required my having to ask for someone to go into Azure and manually execute a forcible unlock. (Thank you again for helping me with that, by the way!) I disabled locking here to prevent that possibility. I'm happy to remove it if there's a compelling argument for doing so, though!

Reply:
I think that's fixing the symptoms rather than the cause, which is forcibly interrupting Terraform while it's holding the lock. We want to Terraform to fail if it can't hold a lock, rather than avoiding asking for one, so we can guarantee state correctness. I'd much rather manually remove the lock every once in a while.

Reply:
Understood! I totally see where you're coming from. Just to make fully sure, as a disclosure: if we run into this situation, deployments to any/all environments will be blocked until someone manually intervenes. There's always the possibility that GH kills off one of its own runners, or some other blip with a system causes this to happen without a human having caused it. As we move toward greater resiliency and additional automation, forcing a lock here introduces a point of failure/hand-touchy. Ideally, we will want to move toward limiting who has Azure SU access as we move forward and/or transition this as it becomes more mature, so that requirement could add a bit more of a burden to operations personnel.

Again, totally fine doing this, and will happily push a commit to do so later today! I want to make sure we're on the same page before I do.

Reply:
I agree with everything you've said. That said, I think I've seen a non-human caused Terraform termination maybe twice ever, so in my mind the likelihood of this being a recurring problem is very low. Additionally, when there's a lock failure, there's an RCA we're going to want to do to make sure this isn't a real problem, and a held lock prevents someone else from updating the infrastructure that may or may not be in a coalesced state in the meantime.

Reply:
Ah, I hadn't considered the RCA need! Thanks for that extra bit of context, and for this really constructive discussion. I've re-enabled the locking on this, per your comments.